Portal/src/Datahub.Functions/AzureConfig.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Datahub.Functions;
+
+internal class AzureConfig
+{
+    private readonly IConfiguration _config;
+    private readonly EmailNotification _emailConfig; 
+
+    public AzureConfig(IConfiguration config)
+    {
+        _config = config;
+        _emailConfig = new EmailNotification();
+        _config.Bind("EmailNotification", _emailConfig);
+    }
+
+    public string LoginUrl => "https://login.microsoftonline.com/";
+    public string ManagementUrl => "https://management.azure.com/";
+    public EmailNotification Email => _emailConfig;
+
+    #region Azure SP
+
+    public string TenantId => _config["TENANT_ID"] ?? "";
+    public string ClientId => _config["FUNC_SP_CLIENT_ID"] ?? "";
+    public string ClientSecret => _config["FUNC_SP_CLIENT_SECRET"] ?? "";
+    public string SubscriptionId => _config["SUBSCRIPTION_ID"] ?? "";
+    
+    #endregion
+
+    #region Storage Notifications
+
+    public string StorageQueueConnection => _config["DatahubStorageConnectionString"] ?? "";
+    public string MaxStorageCapacity => _config["MAX_STORAGE_CAPACITY"] ?? "180000000000"; //"2000000000000";
+    
+    #endregion
+}
```

Comment:
I love this config class, this is what we need for the Datahub config options
cc @ErikApption

Reply:
The idea is to have all the config names in one place. Also validate some configurations, etc.